File: fix_lint.py
with open('src/app/blog/page.tsx', 'r') as f:
    content = f.read()

content = content.replace("<header", "<MedicalDisclaimer />\n      <header")

with open('src/app/blog/page.tsx', 'w') as f:
    f.write(content)

File: fix_lint.mjs
import fs from 'fs';

let content = fs.readFileSync('src/app/blog/[slug]/page.tsx', 'utf8');
content = content.replace("import { getArticleBySlug, getArticleSlugs, getAllArticles } from '@/lib/mdx';", "import { getArticleBySlug, getAllArticles } from '@/lib/mdx';");
fs.writeFileSync('src/app/blog/[slug]/page.tsx', content);
